### PR TITLE
New port: Frozen Bubble

### PR DESCRIPTION
**Pull request by cdeletre**
# New Port for Frozen Bubble

## Game Information
- **Title**: Frozen Bubble
- **URL**: https://github.com/kthakore/frozen-bubble/

### CFW Tests

- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests

- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure

```
.
├── README.md
├── cover.jpg
├── frozen-bubble
│   ├── data
│   ├── fb-server
│   ├── frozen-bubble
│   ├── frozen-bubble.GPL-2.0.LICENSE.txt
│   ├── frozen-bubble.gptk
│   ├── gfx.squashfs
│   ├── icons
│   ├── libs.aarch64
│   ├── locale
│   ├── perl.squashfs
│   └── snd
├── frozen-bubble.sh
├── gameinfo.xml
├── port.json
└── screenshot.jpg
```

## Controls

| Button | Action |
|--|--| 
|D-PAD left/right|aim left/right (P1)|
|D-PAD down|aim center  (P1)|
|D-PAD up|fire (P1)|
|L1|fire (P1)|
|L-Stick up|send malus to top left player (multiplayer only)|
|L-Stick down|send malus to bottom left player (multiplayer only)|
|R-Stick up|send malus to top right player (multiplayer only)|
|R-Stick down|send malus to bottom right player (multiplayer only)|
|R3|send malus to all opponents (multiplayer only)|
|Y/A|aim left/right (P2)|
|B|aim center (P2)|
|X|fire (P2)|
|R1|fire (P2)|
|Start|Enter|
|Select|Esc|
|L2|Y (yes)|
|R2|N (no)|
|L3|toggle music on/off|

### CODE REVIEW

File: ports/frozen-bubble/README.md
Comment (Cebion) [inline ports/frozen-bubble/README.md:1]: Compile Instructions are missing in the README.md

File: ports/frozen-bubble/cover.jpg
Comment (Cebion) [inline ports/frozen-bubble/cover.jpg:1]: Cover should be 640x480 and show gameplay and or a gameplay / coverartmix or gameplay / logo artmix.

File: ports/frozen-bubble/frozen-bubble.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# PORTMASTER: render96ex.zip, render96ex.sh
```
Comment (Cebion) [inline ports/frozen-bubble/frozen-bubble.sh:2]: This line is autogenerated by PortMaster.  Can be removed
```suggestion

```

File: ports/frozen-bubble/frozen-bubble/libs.aarch64/amberelec_ldd.txt
Comment (Cebion) [inline ports/frozen-bubble/frozen-bubble/libs.aarch64/amberelec_ldd.txt:1]: Is this file necessary?

File: ports/frozen-bubble/gameinfo.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./frozen-bubble.sh</path>
```
Comment (Cebion) [inline ports/frozen-bubble/gameinfo.xml:4]: .sh Name needs to be Uppercase. This is the Name that is shown when no Metadata is used.

File: ports/frozen-bubble/port.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "frozen-bubble.zip",
+  "items": [
+    "frozen-bubble.sh",
```
Comment (Cebion) [inline ports/frozen-bubble/port.json:5]: .sh needs to be Upper Case

File: ports/frozen-bubble/screenshot.jpg
Comment (Cebion) [inline ports/frozen-bubble/screenshot.jpg:1]: this is already perfect for cover. So use screenshot.jpg in the gameinfo.xml instead of cover.jpg

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/frozen-bubble/frozen-bubble/libs.aarch64/amberelec_ldd.txt
Comment (cdeletre) [reply ports/frozen-bubble/frozen-bubble/libs.aarch64/amberelec_ldd.txt:1]: Indeed, it's useless now :)

**Comment by cdeletre** [comment]
New commit :)

**Review by Cebion — APPROVED** [review]

**Pull request merged**